routes/all_reports.js

```diff
@@ -0,0 +1,40 @@
+var express = require("express");
+var router = express.Router();
+
+const User = require("../models/user");
+
+/* displays results */
+router.get("/", function (req, res, next) {
+    User.find({}).then(allTheData => {
+        console.log(allTheData);
```

Comment:
Was this intended for testing purposes ? should we leave it since it is WIP for further testing and then remove?